proto/api.proto

```diff
@@ -26,13 +27,14 @@ service APIService {
 
   rpc UpdateAddressField(AddressFieldUpdateRequest) returns (Response) {
     option (google.api.http) = {
-      put:  "/addressbook"
+      put:  "/addressbook/update",
       body: "*"
     };
   }
   rpc DeleteAddressField(AddressFieldRequest) returns (Response) {
     option (google.api.http) = {
-      delete: "/addressbook"
+      put: "/addressbook/delete"
+      body: "*"
```

Comment:
Please avoid of using the body in DELETE requests (however it's possible and may be the only way in some specific cases), see details [here](https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.5).
Instead of that, you may use something similar to
```
delete: "/addressbook/{field.phone}"
```
put, patch or post should not be used for delete purposes as well.